src/registrar/fixtures.py

```diff
@@ -86,6 +86,12 @@ class UserFixture:
             "first_name": "Alysia-Analyst",
             "last_name": "Alysia-Analyst",
         },
+        {
+            "username": "91a9b97c-bd0a-458d-9823-babfde7ebf44",
+            "first_name": "katherine-Analyst",
```

Comment:
```suggestion
            "first_name": "Katherine-Analyst",
```
Was lowercase, not a blocking issue though